_pages/en/target-solution-delivery-model-strategy-areas-of-concerns.md

```diff
@@ -30,353 +32,293 @@ This document will be updated throughout the elaboration of the above strategy.
-What is being discussed is the 10% vs 10x debate.
-When attempting to improve a solution by 10%, generally it is a continuation of previously made software choices and architectural decisions.
-In contrast, if a team is challenged to improve the solution by a factor or 10 (10x) then they will likely rethink the entire solution, which may end up yielding better results.
-These are challenges with the iterative continuous improvement approach (and why innovation is important).
-However, when the argument becomes "are we optimally improving this solution" rather than "did we just waste 100M on a solution that delivered no value to users" then this is judged to be an improvement upon the status quo.
+Current thoughts are on:
 
-TBD. The current thinking is to improve the organization's monitoring ability by linking investments to the health of capabilities, capturing when such investment reaches a threshold to change course and have resources shifting somewhere else.
+- Defining a Target Architecture Vision for legacy transitioning environments.
+This to provide direction and set reasonable expectations for DevOps teams working on non-cloud environments.
+This Target Architecture vision is meant to provide with reference architectures and baked-in requirements in the scope of IT-enabled Projects that touch mainframe programs.
+e.g., an integration layer to expose certain functions and data via publicly available APIs that other IT solutions can consume (i.e., the “[strangler pattern](https://dzone.com/articles/monolith-to-microservices-using-the-strangler-patt)” that you may have heard about).
 
-### 2.11. Risk to enterprise reference data model and standard compliance
+- Identify a pilot project during the implementation of the strategy to test its feasibility.
 
-#### 2.11.1. Summary <!-- omit in toc -->
+### 2.8. Legislative level and mandated deadlines
 
-Moving towards a more decentralized and distributed means of decision making presents a risk that incorrect decisions may occur affecting data management across the enterprise. The risk involved duplication of data, incoherence to data consumers, and degradation of data quality.  Over the years, IITB has spent efforts centralizing certain functions to achieve efficiencies and be able to do more with less.
+#### 2.8.1. Summary <!-- omit in toc -->
 
-In the case of the Data Analytics team, direct access to a database as opposed to an API (exposing data meant for consumption by a User Interface or another functional API) is necessary due to the broad nature of analytics and its need to vast amount of data. An API may be build, per product, for the sole purpose of accessing the database content but:
+The EI world deals with large legislative projects where the work typically starts before gaining official approvals to meet the mandated deadlines.
```

Comment:
This sentence doesn't sound like a concern to me. On the contrary, it seems pretty much aligned with the Strategy.
Or maybe I'm getting it wrong?

But below "how to address the legislative level" is very valid.

Reply:
Indeed. This came from an email by a BSI counterpart. I believe their reasoning is that some of the governance problems the strategy seeks to resolve won't apply since those problems do not necessarily exist for them (e.g. seeking permission to start work). I wanted to still document those concerns as part of the strategy is to not just get permission to start, but to break down large problems into more manageable parts.

Reply:
Good point. I reworded as this (changes in bold)., feel free to edit:

> The EI world deals with large legislative projects **where, in alignment with the spirit of the strategy**, the work typically starts before gaining official approvals to meet the mandated deadlines.
> **Still,** how will this strategy **address the legislative context as a whole**?